test/e2e/nodepool_test.go

```diff
@@ -126,6 +126,14 @@ func TestNodePool(t *testing.T) {
 				}}
 			},
 		},
+		{
+			build: func(ctx context.Context, mgtClient crclient.Client, hostedCluster *hyperv1.HostedCluster, hostedClusterClient crclient.Client, clusterOpts e2eutil.PlatformAgnosticOptions) []NodePoolTestCase {
```

Comment:
@Patryk-Stefanski why did you add it to a separate HostedCluster? I'm just wondering. This has a cost on CI resources and I just noticed this test runs on a dedicated HostedCluster. I wonder if the first (and used for other tests) could be re-used or if there is a reason not to. Thanks

Reply:
IIRC it caused a node rollout that caused other test failures.